static/app/utils/discover/fieldRenderers.tsx

```diff
@@ -650,7 +650,7 @@ const SPECIAL_FIELDS: SpecialFields = {
   user: {
     sortField: 'user',
     renderFunc: data => {
-      if (data.user) {
+      if (data.user?.split) {
```

Comment:
What's this? When does `data.user` not have `.split` if it's defined?

Reply:
I'm re-using this in `attrs` tree which aren't fields, some values I've been testing with logs attrs are sending an `int`